### PR TITLE
Various Azure IoT Hub doc improvements

### PR DESCRIPTION
**Pull request by jtguggedal**
This PR contains several documentation improvements to Azure IoT Hub library and sample, and one change to a shared doc file where nRF9160DK board name is updated.

The improvements are intended to address some relatively frequent issues that users have with credentials and device registration with and without DPS.

### CODE REVIEW

**Review by coderbyheart — APPROVED** [review]

**Review by simensrostad — APPROVED** [review]

File: doc/nrf/libraries/networking/azure_iot_hub.rst [hunk 1]
```diff
@@ -26,17 +26,26 @@ In order to connect to Azure IoT Hub, an Azure account and an Azure IoT Hub inst
...
 The Azure IoT Hub library requires the provisioning of the following certificates and a private key for a successful TLS connection:
 
 1. `Baltimore CyberTrust Root certificate`_ - Server certificate, used to verify the server's certificate while connecting.
-#. Device certificate - generated by the procedures described in `Creating Azure IoT Hub certificates`_ , used by Azure IoT Hub to authenticate the device.
+#. Public device certificate - generated by the procedures described in `Creating Azure IoT Hub certificates`_ , used by Azure IoT Hub to authenticate the device.
 #. Private key of the device.
 
+.. note::
+   The location and name of the public device certificate and private key files that are generated will vary with what method you use for the credential generation.
+   For PowerShell scripts, the device certificate is called ``mydevice-public.pem`` and the private key ``mydevice-private.pem`` and they are located together with the other generated files in the working directory.
```
Comment (peknis) [inline doc/nrf/libraries/networking/azure_iot_hub.rst:42]:
```suggestion
   The location and name of the generated public device certificate and private key files vary depending on the method you use for the credential generation.
   For PowerShell scripts, the device certificate is called :file:`mydevice-public.pem` and the private key :file:`mydevice-private.pem`.
   These files are located  in the working directory with the other generated files.
```

File: doc/nrf/libraries/networking/azure_iot_hub.rst [hunk 2]
```diff
@@ -26,17 +26,26 @@ In order to connect to Azure IoT Hub, an Azure account and an Azure IoT Hub inst
...
+
+   For bash scripts, the public device certificate is called ``new-device.cert.pem`` and are located in a directory called ``certs`` within the script directory.
+   The private key will be called ``new-device.key.pem`` and be located in a directory called ``private`` within the script directory.
```
Comment (peknis) [inline doc/nrf/libraries/networking/azure_iot_hub.rst:45]:
```suggestion
   For bash scripts, the public device certificate is called :file:`new-device.cert.pem` and is located in a directory called :file:`certs` within the :file:`script` directory.
   The private key is called :file:`new-device.key.pem` and located in a directory called :file:`private` within the :file:`script` directory.
```

File: doc/nrf/libraries/networking/azure_iot_hub.rst [hunk 3]
```diff
@@ -26,17 +26,26 @@ In order to connect to Azure IoT Hub, an Azure account and an Azure IoT Hub inst
...
+
+   Note that these names may change if Azure changes their scripts.
```
Comment (peknis) [inline doc/nrf/libraries/networking/azure_iot_hub.rst:47]:
```suggestion
   The file and directory names may change if Azure changes their scripts.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Comment by peknis** [comment]
These changes might be worth a changelog entry.

**Comment by jtguggedal** [comment]
@peknis I've addressed your comments and added a changelog entry

File: doc/nrf/releases/release-notes-changelog.rst [hunk 4]
```diff
@@ -458,6 +458,7 @@ Libraries for networking
   * Added handling of MQTT ping failures and MQTT input failures.
   * Updated the API version used in MQTT connection to Azure IoT Hub to 2020-09-30.
   * Added the :c:func:`azure_iot_hub_dps_reset` function for resetting the DPS information.
+  * Added information in the documenation about which credentials to use and where to find them.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:461]:
```suggestion
  * Added a note about the credentials and their location.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst (hunk 4 above)
Comment (jtguggedal) [reply doc/nrf/releases/release-notes-changelog.rst]: Fixed

**Review by peknis — APPROVED** [review]

**Pull request merged**